scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,69 +2,30 @@
 
 # fail if some commands fails
 set -e
-# show commands
-set -x
-
-ARCH=$(uname -m)
-export CI="openshift"
-if [ "${ARCH}" == "s390x" ]; then
-    make configure-installer-tests-cluster-s390x
-elif [ "${ARCH}" == "ppc64le" ]; then
-    make configure-installer-tests-cluster-ppc64le
-else
-    make configure-installer-tests-cluster
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
 fi
-make bin
-mkdir -p $GOPATH/bin
-make goget-ginkgo
-export PATH="$PATH:$(pwd):$GOPATH/bin"
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
-
-# Copy kubeconfig to temporary kubeconfig file
-# Read and Write permission to temporary kubeconfig file
-TMP_DIR=$(mktemp -d)
-cp $KUBECONFIG $TMP_DIR/kubeconfig
-chmod 640 $TMP_DIR/kubeconfig
-export KUBECONFIG=$TMP_DIR/kubeconfig
 
-# Login as developer
-odo login -u developer -p password@123
+##### These are varialbes used by ci-firewall as one of the ways to get its parameters
+# If AMQP_URI is not set by the time we reach here, show error message and exit.
+export AMQP_URI=${AMQP_URI:?"Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"}
+export JOB_NAME="odo-pr-build"
```

Comment:
`odo-pr-build` seems so generic, isn't it? How do we differentiate minikube and openshift pr build. Can we say it as `odo-ocp-pr-build`?